### PR TITLE
Module Release 0.1.0

### PR DESCRIPTION
**Pull request by ghost**
- Developer(s): @sergeysikachev @sergeypalkin @alex-galych 

- Peer Reviewer:

- Ticket: URL_HERE

- Project PR: URL_HERE

- Academy PR: URL_HERE (for feature documentation and if needed migration guides and project integration guide)

#### Please confirm

- [ ] No new OS components - or they have been approved by legal department
- [ ] All new or heavily changed classes get an "A" rating (Scrutinizer), except Facades, Factories, QueryContainers and Tests

#### Documentation

- [ ] Functional documentation provided or in progress?
- [ ] Integration guide for projects provided or not needed?
- [ ] Migration guides for all contained majors provided or not needed?

#### Release Table

   Bundle                | Release Type         | Constraints         |
   :--------------------- | :------------------------ | :--------------------- |
   Ratepay               |       minor                 |                       |

#### Release Notes

- Fix error connected with product with warranty in cart.
- Descriptor saving to spy_payment_ratepay table.
- Added environment configuration to Ratepay unit suite.
- Fixed broken place order logic with applied discounts.

-----------------------------------------

#### Bundle Ratepay

_**Major:** Backwards-compatible bug fix_

Def of done (by responsible developer):
- [ ] All changes are backward-compatible. Outdated code is marked as deprecated
- [ ] The change is isolated and not mixed with any cleanups or other changes
- [ ] New code fits to the architecture rules
- [ ] All new or changed business logic is covered by functional tests (in Zed business layer)

##### Change log

Bugfixes

- Module is moved to separate repository from core.
- Fix error connected with product with warranty in cart.
- Descriptor saving to spy_payment_ratepay table.
- Added environment configuration to Ratepay unit suite.
- Fixed broken place order logic with applied discounts.

### CODE REVIEW

**Comment by dereuromark** [comment]
@sergeysikachev Your github user account on your dev machine / VM is not set up properly.

**Comment by ghost** [comment]
@dereuromark, Thank you, I have this repo outside the VM and I forgot to set up git name and email there. Fixed.

File: .scrutinizer.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+build:
+    environment:
+        php:
+            version: 7.1
```
Comment (dereuromark) [inline .scrutinizer.yml]: Arent those files 2 space indentation? 
Please also check if .editorconfig is set correctly.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+notifications:
+  email: false
+
+sudo: required
+
+matrix:
+  fast_finish: true
+  include:
+     - php: "7.0"
```
Comment (dereuromark) [inline .travis.yml]: 7.1?

File: assets/Yves/package.json [hunk 3]
```diff
@@ -0,0 +1,8 @@
+{
+  "private": true,
+  "name": "spryker-yves-ratepay",
+  "version": "1.0.0",
+  "dependencies": {
+    "jquery": "~2.2.0"
```
Comment (alex-galych) [inline assets/Yves/package.json]: Has to be updated to ~3.0

**Review by alex-galych — CHANGES_REQUESTED** [review]

**Comment by dereuromark** [comment]
Might also require new releases of Spryker core now, as well as e.g.

- "spryker/session": "^3.0.0 || ^4.0.0",

compatibility

**Comment by dereuromark** [comment]
Also please be aware of new fixes , e.g.
- https://github.com/spryker/ratepay/releases/tag/0.6.11
- https://github.com/spryker/ratepay/releases/tag/0.6.12

**Review by alex-galych — APPROVED** [review]

**Pull request merged**